db/migrate/20170720142145_add_deb_release_parameters_to_repositories.rb

```diff
@@ -0,0 +1,7 @@
+class AddDebReleaseParametersToRepositories < ActiveRecord::Migration
+  def change
+    add_column :katello_repositories, :deb_releases, :string, limit: 255
+    add_column :katello_repositories, :deb_components, :string, limit: 255
+    add_column :katello_repositories, :deb_architectures, :string, limit: 255
```

Comment:
Why create these, cant it use the columns already in this table? e.g `distribution_arch`, `distribution_version` ect..